src/nft/components/bag/BagFooter.tsx

```diff
@@ -22,13 +22,17 @@ import { switchChain } from 'utils/switchChain'
 
 import * as styles from './BagFooter.css'
 
+const FooterContainer = styled.div`
```

Comment:
There seems to be some extra top padding with the button warnings
<img width="337" alt="Screen Shot 2022-12-13 at 08 53 34 " src="https://user-images.githubusercontent.com/11512321/207388948-f84973fc-83b0-46a5-88bf-9f618ed69112.png">